app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -244,6 +245,29 @@ gql`
   }
 `
 
+gql`
+  query sendBitcoinLimits($isWithdrawal: Boolean!, $isInternalSend: Boolean!) {
```

Comment:
I think it would have been  practice to separate the query in 2 independant queries here

Reply:
any reason?

Reply:
it favors caching afair

Reply:
not relevant for us at this stage because we don't have caching enabled at the HTTP layer